aviary/modules/assembly/scripts/spades_assembly.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import os
+import sys
+from typing import List
+
+def spades_asssembly(
+    input_fastq: str,
+    input_long_reads: str,
+    output_fasta: str,
+    output_spades_folder: str,
+    max_memory: int,
+    threads: int,
+    kmer_sizes: List[int],
+    tmp_dir: str,
+    long_read_type: str,
+    log: str):
+
+    '''
+    Assemble short reads and long reads (if any) using spades
+    :param input_fastq: short reads fastq file
+    :param input_long_reads: long reads fastq file
+    :param output_fasta: output fasta file
+    :param output_spades_folder: output spades folder
+    :param max_memory: maximum memory to use
+    :param threads: number of threads
+    :param tmpdir: temporary directory
+    :param kmer_sizes: list of kmer sizes
+    :param long_read_type: type of long reads
+    :param log: log file
+    :return:
+    '''
+    if tmp_dir:
+        tmp_dir_arg = f"--tmp-dir {tmp_dir}"
+    else:
+        try:
+            tmp_dir = os.environ["TMPDIR"]
+            tmp_dir_arg = f"--tmp-dir {tmp_dir}"
+        except KeyError:
+            tmp_dir_arg = ""
+    
+    if os.path.exists("data/spades_assembly/tmp"):
+        subprocess.call("rm -rf data/spades_assembly/tmp",shell=True)
+    # remove existing temporary directory
+    minimumsize=500000
+    actualsize = int(subprocess.check_output('stat -c%s data/short_reads.filt.fastq.gz', shell=True))
+    # check if directory exists
+    if  os.path.exists("data/spades_assembly"):
+        # resume previous assembly
+        command = f"spades.py --restart-from last --memory {max_memory} -t {threads} " \
+                f"-o data/spades_assembly -k {kmer_sizes}  {tmp_dir_arg}" 
+        # run cmd
+        with open(log, 'a') as logf:
+            logf.write(f"Queueing command {command}\n")
+            subprocess.run(command.split(), stdout=logf, stderr=subprocess.STDOUT)        
+        subprocess.call("cp data/spades_assembly/scaffolds.fasta data/spades_assembly.fasta",shell=True)
+    elif actualsize >= minimumsize:
+        if long_read_type in ["ont","ont_hq"]:
+            command = f"spades.py --checkpoints all --memory {max_memory} --meta --nanopore {input_long_reads} --12 {input_fastq} "\
+                f"-o data/spades_assembly -t {threads}  -k {kmer_sizes} {tmp_dir_arg} "       
+        elif long_read_type == "pacbio":
```

Comment:
Why are you looking for "pacbio", we should use "pacbio" mode when the user supplies "rs", "sqq", "ccs", or "hifi". "pacbio" isn't even an option? 

This should really just be an else statement as it was in the original script.